README.md

```diff
@@ -36,6 +36,7 @@ The package can be installed as:
     username: "your.name@your.domain", # or {:system, "SMTP_USERNAME"}
     password: "pa55word", # or {:system, "SMTP_PASSWORD"}
     tls: :if_available, # can be `:always` or `:never`
+    tls_versions: [:"tlsv1", :"tlsv1.1", :"tlsv1.2"], # additional details: http://erlang.org/doc/man/ssl.html
```

Comment:
I would suggest to replace the comment ("additional_details...") to stay consistent with other values. 
Something like:

```elixir
tls_versions: [:"tlsv1", :"tlsv1.1", :"tlsv1.2"],  # or {:system, "TLS_VERSIONS"} (comma separated values)
```

Again, to keep a consistent documentation, could you copy your new line to the `lib/bamboo/adapters/smtp_adapter.ex` documentation? 
If you think it can help, we could also add a line there about what `tls_versions` is and also a link to the GenSMTP documentation instead of the Erlang one (https://github.com/Vagabond/gen_smtp#options). It might be easier to understand than the SSL documentation.

What do you think?